### PR TITLE
add trolling

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
troll mask :trollface:

## Why / Balance
:trollface:

## Technical details
:trollface:

## Media
what it looks like:
:trollface:

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
:trollface:

**Changelog**
:cl:
- add: Added a troll mask to the AutoDrobe when the manager wire is cut, presumably for trolling.

### CODE REVIEW

**Comment by Just-a-Unity-Dev** [comment]
pro

**Comment by deltanedas** [comment]
it looks fuckeed with a hat, not sure how to make it draw over it

**Comment by asperger-sind** [comment]
based pr we need this shit!!!!!!!!!!!!!!

**Comment by VektorZ1** [comment]
:trollface:

File: Resources/Prototypes/Entities/Clothing/Masks/masks.yml [hunk 1]
```diff
@@ -194,6 +194,17 @@
   - type: BreathMask
   - type: IdentityBlocker
 
+- type: entity
+  parent: ClothingMaskJoy
+  id: ClothingMaskTroll
+  name: troll mask
+  description: Problem?
```
Comment (ficcialfaint) [inline Resources/Prototypes/Entities/Clothing/Masks/masks.yml:201]: :trollface: 
```suggestion
  description: real pro
```

Comment (deltanedas) [reply Resources/Prototypes/Entities/Clothing/Masks/masks.yml:201]: fake noob

**Comment by moonheart08** [comment]
no

**Comment by JoeHammad1844** [comment]
1984

**Pull request closed without merging**